core/dropdowndiv.js

```diff
@@ -293,10 +293,9 @@ exports.showPositionedByField = showPositionedByField;
  */
 const getScaledBboxOfBlock = function(block) {
   const blockSvg = block.getSvgRoot();
-  const bBox = blockSvg.getBBox();
   const scale = block.workspace.scale;
-  const scaledHeight = bBox.height * scale;
-  const scaledWidth = bBox.width * scale;
+  const scaledHeight = block.height * scale;
```

Comment:
Did you test this change? It won't be exercise during deletion.

Reply:
Ah no I forgot. It's actually not exercised at all because we never call `showPositionedByBlock`. So I'll have to write something special to test it.